zvariant_derive/src/value.rs

```diff
@@ -130,7 +136,12 @@ fn impl_struct(
                     let (fields_init, entries_init): (TokenStream, TokenStream) = fields
                         .iter()
                         .map(|field| {
+                            let FieldAttributes { rename } =
+                                FieldAttributes::parse(&field.attrs).unwrap_or_default();
                             let field_name = field.ident.to_token_stream();
+                            let dict_name =
```

Comment:
This needs a better name too. :) shadowing `field_name` would be fine if it's not used again later.

Reply:
Changed it to key name as it is for reference the keys in the dict.